tests/L0/run_test.py

```diff
@@ -43,12 +45,32 @@ def parse_args():
         default=DEFAULT_TEST_DIRS,
         help="select a set of tests to run (defaults to ALL tests).",
     )
+    parser.add_argument(
+        "--xml-report",
+        action="store_true",
+        help="pass this argument to get a junit xml report. (requires `xmlrunner`)",
+    )
     args, _ = parser.parse_known_args()
     return args
 
 
-def main(args):
-    runner = unittest.TextTestRunner(verbosity=2)
+def main(args: argparse.Namespace) -> None:
+    xml_output, xml_filename = None, None
+    test_runner_kwargs = {"verbosity": 2}
+    Runner = unittest.TextTestRunner
+    if args.xml_report:
+        import xmlrunner
+        from datetime import date  # NOQA
+        this_dir = os.path.abspath(os.path.dirname(__file__))
+        xml_filename = os.path.join(
+            this_dir,
+            f"""{date.today().strftime("%y%m%d")}.xml""",
+        )
+        xml_output = open(xml_filename, "w")
```

Comment:
Locally only "w" worked, but not in NGC pytorch container

```suggestion
        xml_output = open(xml_filename, "wb")
```